scania-digital-design-system/tegel
# test(text-field): default test case

**Describe pull-request**  
Adding unit tests for default case of Text-Field component

**Solving issue**  
Fixes: [CDEP-3043](https://tegel.atlassian.net/browse/CDEP-3043)

**How to test**  
1. Check default test case of tds-text-field [component](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/Documents/Unit%20test%20scenarios.xlsx?d=wca48bf6d92cf4e8c8f456337e2002ec4&csf=1&web=1&e=4FrlwX&nav=MTVfezA2MzcyM0JBLUEyNkQtNEQyMy1BNUUwLTlFMzJCQ0E1OEQ4OX0)
2. Check if added test look OK
3. Run tests and see if all of them pass

**Checklist before submission**

**Additional context**  
Other test cases will be added in separate commits. To avoid a long review process and better use short development time I decided to chop it in smaller chunks. 

I have increased pixel tolerance in table tests as usually they fail when 1% of total number of pixel is off different. It tends to happen in components with dropdowns or some kind of change on click etc, here it is filtering and expandable rows...


[CDEP-3043]: https://tegel.atlassian.net/browse/CDEP-3043?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
